DQMOffline/Trigger/python/JetMETPromptMonitoring_Client_cff.py

```diff
@@ -0,0 +1,172 @@
+        "effic_pfjetEta_HEM17       'Jet efficiency vs #eta (HEM17); PF Jet phi [rad]; efficiency' pfjetEta_HEM17_numerator       pfjetEta_HEM17_denominator",
+
+        "effic_pfjetpT_HEP18          'Jet pT turnON (HEP18);            PFJet(pT) [GeV]; efficiency'     pfjetpT_HEP18_numerator          pfjetpT_HEP18_denominator",
+        "effic_pfjetpT_HEP18_pTThresh 'Jet pT turnON (HEP18);            PFJet(pT) [GeV]; efficiency'     pfjetpT_pTThresh_HEP18_numerator pfjetpT_pTThresh_HEP18_denominator",
+        "effic_pfjetPhi_HEP18       'Jet efficiency vs #phi (HEP18); PF Jet #phi [rad]; efficiency' pfjetPhi_HEP18_numerator       pfjetPhi_HEP18_denominator",
+        "effic_pfjetEta_HEP18       'Jet efficiency vs #eta (HEP18); PF Jet #eta [rad]; efficiency' pfjetEta_HEP18_numerator       pfjetEta_HEP18_denominator",
+
+        "effic_pfjetpT_HF          'Jet pT turnON (HF);            PFJet(pT) [GeV]; efficiency'     pfjetpT_HF_numerator          pfjetpT_HF_denominator",
+        "effic_pfjetpT_HF_pTThresh 'Jet pT turnON (HF);            PFJet(pT) [GeV]; efficiency'     pfjetpT_pTThresh_HF_numerator pfjetpT_pTThresh_HF_denominator",
+        "effic_pfjetPhi_HF       'Jet efficiency vs #phi (HF); PF Jet phi [rad]; efficiency' pfjetPhi_HF_numerator       pfjetPhi_HF_denominator",
+        "effic_pfjetEta_HF       'Jet efficiency vs #eta (HF); PF Jet phi [rad]; efficiency' pfjetEta_HF_numerator       pfjetEta_HF_denominator",
+        ## 2D Eff
+        "effic_pfjetEtaVsPhi       'Jet efficiency vs #eta and #phi; PF Jet #eta; #phi' pfjetEtaVsPhi_numerator       pfjetEtaVsPhi_denominator",
+        "effic_pfjetEtaVsPhi_HB    'Jet efficiency vs #eta and #phi(HB); PF Jet #eta; #phi' pfjetEtaVsPhi_HB_numerator       pfjetEtaVsPhi_HB_denominator",
+        "effic_pfjetEtaVsPhi_HE    'Jet efficiency vs #eta and #phi(HE); PF Jet #eta; #phi' pfjetEtaVsPhi_HE_numerator       pfjetEtaVsPhi_HE_denominator",
+        "effic_pfjetEtaVsPhi_HF    'Jet efficiency vs #eta and #phi(HF); PF Jet #eta; #phi' pfjetEtaVsPhi_HF_numerator       pfjetEtaVsPhi_HF_denominator",
+        "effic_pfjetEtaVsPhi_HEM17 'Jet efficiency vs #eta and #phi(HEM17); PF Jet #eta; #phi' pfjetEtaVsPhi_HEM17_numerator       pfjetEtaVsPhi_HEM17_denominator",
+        "effic_pfjetEtaVsPhi_HEP17 'Jet efficiency vs #eta and #phi(HEP17); PF Jet #eta; #phi' pfjetEtaVsPhi_HEP17_numerator       pfjetEtaVsPhi_HEP17_denominator",
+        "effic_pfjetEtaVsPhi_HEP18 'Jet efficiency vs #eta and #phi(HEP18); PF Jet #eta; #phi' pfjetEtaVsPhi_HEP18_numerator       pfjetEtaVsPhi_HEP18_denominator",
+        "effic_pfjetEtaVsPhi_HE_p 'Jet efficiency vs #eta and #phi(HE_p); PF Jet #eta; #phi' pfjetEtaVsPhi_HE_p_numerator       pfjetEtaVsPhi_HE_p_denominator",
+        "effic_pfjetEtaVsPhi_HE_m 'Jet efficiency vs #eta and #phi(HE_m); PF Jet #eta; #phi' pfjetEtaVsPhi_HE_m_numerator       pfjetEtaVsPhi_HE_m_denominator",
+    ),
```

Comment:
Dear Mia,

For resolution, I know Simon draw the resolution with hltjet.pt()/genjet.pt().
In this case, I know he uses the prompt reco jet in data, so that I am not sure he could use genjet.
How can he define the resolution in this case?

As far as I know, we could draw the jet  pT or energy distribution and calculate rms/mean since the response of calorimeter to jets would not be Gaussian.
This will be the jet energy resolution in calorimetry.

If I missed something, please let me know.

Thank you,
Sehwook